bundles/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/handler/HueBridgeHandler.java

```diff
@@ -1039,6 +1039,8 @@ private void notifyGroupSceneUpdate(List<Scene> scenes) {
     public Collection<ConfigStatusMessage> getConfigStatus() {
         // The bridge IP address to be used for checks
         // Check whether an IP address is provided
+        hueBridgeConfig = getConfigAs(HueBridgeConfig.class);
```

Comment:
Fixes an NPE because `initialize()` will no be called anymore without missing or wrong configuration parameters.

Reply:
Do you mean this will be the case for any binding? This will be an absolute terrible breaking change. We certainly have many and many bindings getting the configuration in the initialize method.

Reply:
It‘s only an issue if methods in the thing handler are called even if the thing is not initialized AND these methods depend on values set in initialize. IMO this is a faulty behavior.

Reply:
Yes I understood that later and deleted my comment.